p2p/src/authenticated/discovery/actors/tracker/actor.rs

```diff
@@ -772,12 +684,11 @@ mod tests {
                 mut oracle,
                 ip_namespace,
                 tracker_pk,
-                cfg,
                 ..
             } = setup_actor(context.clone(), cfg_initial);
 
             let ts_new = context.current().epoch_millis();
-            let ts_old = ts_new.saturating_sub(cfg.synchrony_bound.as_millis() as u64 / 2);
```

Comment:
I assume it might be less brittle to keep the value tied to `cfg.synchrony_bound` rather than having it be an "unrelated" constant

Edit: ah, I see that there might not be a `cfg.synchrony_bound` anymore...